File: backend/internal/reporting/enum.go
package reporting

type DocumentType string

var (
	EmploymentProof                DocumentType = "employment_proof"
	SelfEmploymentProof            DocumentType = "self_employment_proof"
	StudentProof                   DocumentType = "student_proof"
	FiledTaxInDesiredLocationProof DocumentType = "filed_tax_in_desired_location_proof"
	ResidesInDesiredLocationProof  DocumentType = "resides_in_desired_location_proof"
)

func (dT DocumentType) IsValid() bool {
	switch dT {
	case EmploymentProof,
		SelfEmploymentProof,
		StudentProof,
		FiledTaxInDesiredLocationProof,
		ResidesInDesiredLocationProof:
		return true
	}
	return false
}

File: apps/backend/src/shared/types/modules/reporting/enum.py
from enum import StrEnum


class DOCUMENT_TYPE(StrEnum):
    employment_proof = "employment_proof"
    self_employment_proof = ("self_employment_proof",)
    student_proof = "student_proof"
    filed_tax_in_desired_location_proof = "filed_tax_in_desired_location_proof"
    resides_in_desired_location_proof = "resides_in_desired_location_proof"


class CHILD_DOCUMENT_TYPE(StrEnum):
    disability_proof = "disability_proof"
